### PR TITLE
feat: add tile sdk demo

### PR DESCRIPTION
**Pull request by bryans99**
Also:
1. update @testing-library/jest-dom package
2. add assertion to tile-visualization test

### CODE REVIEW

File: react/javascript/tile-sdk/src/components/Inspector/components/TileHostData/TileHostData.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2022 Looker Data Sciences, Inc.
```
Comment (jkaster) [inline react/javascript/tile-sdk/src/components/Inspector/components/TileHostData/TileHostData.js:5]: 2022?

File: react/javascript/tile-sdk/src/components/Inspector/components/VisualizationData/VisualizationData.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2022 Looker Data Sciences, Inc.
```
Comment (jkaster) [inline react/javascript/tile-sdk/src/components/Inspector/components/VisualizationData/VisualizationData.js:5]: 2022

**Review by jkaster — DISMISSED** [review]
There's some 2022 license headers but they may be older?

File: react/javascript/tile-sdk/src/components/Inspector/components/VisualizationData/VisualizationData.js (hunk 2 above)
Comment (bryans99) [reply react/javascript/tile-sdk/src/components/Inspector/components/VisualizationData/VisualizationData.js:5]: Dammit. Forgot!

**Review by jkaster — APPROVED** [review]

**Pull request merged**